contrib/http_util/src/main/java/io/opencensus/contrib/http/HttpServerHandler.java

```diff
@@ -42,8 +42,8 @@
  * @since 0.18
```

Comment:
Forgot to modify since. Better to do this in a separate PR and remove all todos like this.

Reply:
ok.